_data/developer.yml

```
@@ -422,10 +422,12 @@ websites:
       doc: https://pushover.net/faq#security-2fa
 
     - name: PyPI
-      url: https://pypi.python.org/
-      twitter: pypi
+      url: https://pypi.org/
       img: pypi.png
-      tfa: No
+      tfa: Yes
+      software: Yes
+      hardware: Yes
+      doc: https://pypi.org/help/#2fa
```

@brainwane It seems that by now #2fa is no longer in the help list but #twofa - is that correct?
```suggestion
      doc: https://pypi.org/help/#twofa
```

---

You're right @kmpoppe -- thanks for noticing that! Yes, that URL changed.

---

@brainwane I'm not allowed to commit that change into your repository, would you please be so kind as to do that yourself?

---

Wish I could, @kmpoppe - I think @DanNixon has to!